src/gencost/waterfall.py

```diff
@@ -1746,7 +1819,10 @@ def get_gf923_by_generator(self):
                     "fuel_consumed_mmbtu": "mmbtu",
                 }
             )
-            .pivot_table(
+        )
+
+        if counterfactuals is False:
```

Comment:
Two very minor points here. 
1. The preferred way to branch using a boolean is to do `if x:...` or `if not x:...` depending on if you want to enter the block if x is True or False respectively.
2. In cases like this where you will return from inside the if-block, I think its usually easier to read if youdo the simple variant first and then skip the else. So something like this...

```python

def do(x, y)
    if x:
        return y
    y = y + 5
    return y
```

Reply:
gotcha, I remember us discussing this before - fixed!